stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -220,5 +222,15 @@ interface StripeIntent : StripeModel {
                 ) : Parcelable
             }
         }
+
+        @Parcelize
+        object BlikAuthorize : NextActionData() {
```

Comment:
@mshafrir-stripe 

This falls into the [else](https://github.com/stripe/stripe-android/blob/638aa9f657e441a81fb18f74ea46d46d226c7034/stripe/src/main/java/com/stripe/android/StripePaymentController.kt#L584) case in StripPaymentController as the SDK doesn't need to do anything upon seeing `"next_action": {
    "type": "blik_authorize"
  },
Is this correct behavior?

Reply:
That seems fine for now.